reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+
+The authors then discuss an 'increase in firmness'.
+Logically, this would be the Bayes factor, not the posterior probability of the hypothesis, that the authors are talking about.
+The Bayes factor is a continuous measure of how much we update our belief.
+If we want to argue for any function of preregistration from a Bayesian perspective, it must be focused on the Bayes factor.
+The authors then say 'If a hypothesis is more probable to begin with, observing evidence in its favor will result in a hypothesis that is more strongly confirmed, all else being equal'.
+
+> We explicitly consider the whole space of possible measures. From the revised paper:
+> *The goal is therefore to reason about the space of possible measures researchers might apply.*
+> *However, since any measure fulfilling the statistical relevancy condition increases monotonically with an increase in posterior probability $P(H|E)$, we might well take it to illustrate our reasoning.*
+> Later in the paper: *Of course, the assigned probabilities and the distribution $Q$ vary from study to study and researcher to researcher (and even the measure of confirmation), but we can illustrate the effect of uncertainty with an example.*
+
+This statement is not in line with the earlier statement about an 'increase in firmness'.
+Of course, if a hypothesis is highly probable (the Stroop effect), any evidence will end up with a higher posterior probability that if the prior probability was low.
+But the same evidence would lead to a greater 'increase in firmness' (or a higher Bayes factor) if the prior was lower.
+The same data would lead to a much greater update in belief.
+The authors dismiss the prior probability of the hypothesis as it 'is nothing our study design can change'.
+However, this is not true.
+Although the authors only discuss inflating the alpha level in their paper, one function of preregistration is to prevent HARKing, or hypothesizing after the results are true.
+Of course, if one comes up with a hypothesis after looking at the data, the 'prior' is 1: We already know the hypothesis will be supported by the data.
+
+> We included a section called "Hacking, harking, and other harms", which hopefully makes our thinking on these things clearer.
+
+HARKing is a major reason the field has moved towards preregistration, but the term 'HARKing' is not even mentioned in the article.
+The reason for this is that the arguments the authors provide can not deal with HARKing.
+This is obviously a limitation, compared to existing arguments for preregistration, and it is incorrectly ignored in the article.
+As Mayo (2018) explains, the problem with HARKing is that it leads to a non-severe test:
+The data have no probability of falsifying the claim.
+This error-statistical perspective seems to me to be much more fruitful than a Bayesian approach.
+
+> As stated above, we come to the same conclusions despite being inductionists.
```

Comment:
Again, we agree that there are different perspectives on inference and statistical testing.
While the math is quite similar, they take fundamentally different positions and interpretations of the objects involved (e.g., probabilities, hypotheses, ...). 
Importantly, we come mostly to the same conclusions as people subscribing to an error-statistical perspective yet we believe that the Bayesian perspective provides a general perspective which does not limit ourselfs to an inductionist perspective.

Reply:
not sure why "inductionist" was brought up here (we agreed, this is a difficult term, too). Maybe discuss this?